blkn/src/Input/MaybeError.js

```diff
@@ -0,0 +1,16 @@
+// @flow
```

Comment:
What about exporting this component from a more general path? Eg. `blkn/src/Errors/MaybeErrorLine.js`.?
I'm using exactly this logic and return content for `Select` component as well.

Reply:
One of you should make it even more general, see info message line zpl.io/VDDzogO.

Reply:
You can make it as a separated PR, to not implement it twice.

Reply:
Sure, how could we name this? `FeedbackLine`, `InputFeedback`?

Reply:
Whatever you find as common practice for forms. Feedback is ok, I also found Hint.